access/views.py

```diff
@@ -59,10 +68,13 @@ def publish(request):
         return HttpResponse(status=405)
 
     if "course_id" not in request.POST:
+        SecurityLog.info(request, f"PUBLISH")
```

Comment:
Does the log message describe if the request was accepted or declined?

Reply:
Good now? Sorry about the rebase, the changes were pretty big. Only the views.py and log.py files changed.